lib/src/ioctl/frameintervals.rs

```diff
@@ -0,0 +1,47 @@
+use std::os::unix::io::AsRawFd;
+use thiserror::Error;
+
+use crate::{bindings, PixelFormat};
+
+pub trait FrameSize {
+    fn from(input: bindings::v4l2_frmivalenum) -> Self;
+}
+
+impl FrameSize for bindings::v4l2_frmivalenum {
+    fn from(input: bindings::v4l2_frmivalenum) -> Self {
+        input
+    }
+}
+
+#[doc(hidden)]
+mod ioctl {
+    use crate::bindings::v4l2_frmivalenum;
+    nix::ioctl_readwrite!(vidioc_enum_frameintervals, b'V', 75, v4l2_frmivalenum);
+}
+
+#[derive(Debug, Error)]
+pub enum FrameSizeError {
```

Comment:
Same here, I'd expect `FrameIntervalError` but maybe I am missing the intended logic.

Reply:
No, you are right... a copy&paste problem, thanks for checking!